README.md

```diff
@@ -58,6 +58,26 @@ Here is an example command to let Gradle add all dependency artifacts to your `v
 gradle --refresh-dependencies --write-verification-metadata sha256 --write-locks dependencies
 ```
 
```

Comment:
I think we could give this section a title of its own, e.g. `Detached Configurations` (can't think of anything better at the moment - may be more descriptive...).